src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -91,6 +91,29 @@ protected function doSend(MessageInterface $message): SentMessage
             unset($options['attach']);
         }
 
+        if(isset($options['external'])) {
+
+            $uploadedMedia = $this->uploadMedia(
+                [
+                    [
+                        'file' => $options['external']['thumb'],
+                        'description' => $options['external']['description']
+                    ]
+                ]
+            );
+
+            $options['record']['embed'] = [
+                '$type' => 'app.bsky.embed.external',
+                'external' => [
+                    'uri' => $options['external']['uri'],
+                    'title' => $options['external']['title'],
+                    'description' => $options['external']['description'],
+                    'thumb' => current($uploadedMedia)['image'],
```

Comment:
We reuse the same method as for adding media, but we can only use one image in our case, so we take the first image that comes along.

Reply:
Can we avoid `current()` and take the first one instead? 

```suggestion
                    'thumb' => $uploadedMedia[array_key_first($uploadedMedia)]['image'],
```

Reply:
Done :)